integrations/discord/save.go

```diff
@@ -0,0 +1,49 @@
+package discord
+
+import (
+	"net/http"
+	"strings"
+
+	"go.uber.org/zap"
+
+	"go.autokitteh.dev/autokitteh/sdk/sdkintegrations"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+const (
+	headerContentType = "Content-Type"
+	contentTypeForm   = "application/x-www-form-urlencoded"
+)
+
+var botTokenVar = sdktypes.NewSymbol("bot_token")
```

Comment:
Generally (with a few exceptions): I use `CamelCase` in connection var names, instead of `snake_case` (even though URL params are `snake_cased`)